schema-admin.graphql

```diff
@@ -487,6 +487,97 @@ input CreateApprovedCuratedCorpusItemInput {
     scheduledSurfaceGuid: ID
 }
 
+"""
+Input data for loading an Approved Item via an automated process and optionally scheduling
+this item to appear on a Scheduled Surface.
+"""
+input LoadApprovedCuratedCorpusItemInput {
+    """
+    The GUID of the corresponding Prospect ID. Will be empty for manually added items.
+    """
+    prospectId: ID
+    """
+    The URL of the Approved Item.
+    """
+    url: Url!
+    """
+    The title of the Approved Item.
+    """
+    title: String!
+    """
+    The excerpt of the Approved Item.
+    """
+    excerpt: String!
+    """
+    The outcome of the curators' review of the Approved Item.
+    """
+    status: CuratedStatus!
+    """
+    What language this item is in. This is a two-letter code, for example, 'en' for English.
+    """
+    language: String!
+    """
+    The name of the online publication that published this story.
+    """
+    publisher: String!
+    """
+    The image URL for this item's accompanying picture.
+    """
+    imageUrl: Url!
+    """
+    A topic this story best fits in.
+    Temporarily a string value that will be provided by Prospect API, possibly an enum in the future.
+    """
+    topic: String!
```

Comment:
just calling out, for some items - the `topics` are null. we will pass them as `imported`

Reply:
Sorry could you clarify @sri-kirsh? Will topics be sent to Curated Corpus API as `null` or as a string value of `imported`?

Reply:
I think what Sri means is that the topics that have `null` as a value, we will map it to `imported` in the data mapping process. So this could stay as a required field

Reply:
ya agree on what herraj mentioned - in the mutation input, we will set the `topic` as `imported` when the topics are null in the legacy database. 

topic will be a required field

Reply:
what's our plan for this `topic` value? are we going to add it to the acceptable values in the API? will this introduce any issues when viewing/editing these items in the tool?

Reply:
this conversation is happening in slack: https://pocket.slack.com/archives/CTLKW5WCF/p1646934876602779

Reply:
note: this decision would not be blocking this PR as the `topics` are anyway string and would be a mandatory field in the schema in either case.

Reply:
i have thrown a wrench into this plan by proposing we instead allow `null` on this database field. ([slack thread](https://pocket.slack.com/archives/CTLKW5WCF/p1647014269434169?thread_ts=1646934876.602779&cid=CTLKW5WCF).) if folks are all on-board/no one sees an issue, we can drop this input.

Reply:
changed it as a nullable field